newm-server/src/main/kotlin/io/newm/server/features/idenfy/IdenfyRoutes.kt

```diff
@@ -28,12 +32,17 @@ fun Routing.createIdenfyRoutes() {
         }
 
         post("callback") {
-            repository.processSessionResult(
-                receiveAndVerify(
-                    signatureHeader = "Idenfy-Signature",
-                    key = key
+            try {
```

Comment:
Can you add the same try/catch around the `get("session")` call?

Reply:
Done